doc: suggest supported Kubernetes version (#804)
longhorn/website: content/docs/1.6.0/deploy/important-notes/index.md

### Supported Kubernetes Versions

> Please ensure your Kubernetes cluster is at least v1.21. However, we suggest upgrading your Kubernetes cluster to a version between v1.26 and v1.28 before upgrading to Longhorn v{{< current-version >}}. These specific Kubernetes versions have been tested with Longhorn v{{< current-version >}}.

Reviewer: ```suggestion
Ensure that the Kubernetes version running in your cluster is v1.21 or later. Before upgrading Longhorn to v{{< current-version >}}, upgrade your Kubernetes cluster to a version between v1.26 and v1.28. These specific Kubernetes versions have been tested with Longhorn v{{< current-version >}}.
```

> Ensure that the Kubernetes version running in your cluster is v1.21 or later. Before upgrading Longhorn to v{{< current-version >}}, upgrade your Kubernetes cluster to a version between v1.26 and v1.28. These specific Kubernetes versions have been tested with Longhorn v{{< current-version >}}.

Reviewer 2: @c3y1huang we decided to have at least 1.21 still, right? So asking users to upgrade to 1.26 ~ 1.28 is not right. Also, as per the discussion, we need to describe that the actively supported K8s versions will be recommended in https://longhorn.io/docs/1.6.0/best-practices/ because of the EOL of upstream K8s versions.

Author: Updated. PTAL.
